soap/soap_connector.bal

```diff
@@ -32,10 +32,27 @@ public type SoapConnector object {
         P{{request}} Request to be sent
         R{{}} If success, returns the response object, else returns `SoapError` object
     }
-    public function sendSoapRequest(string path, SoapRequest request) returns (SoapResponse|SoapError);
+    public function sendReceive(string path, SoapRequest request) returns (SoapResponse|SoapError);
```

Comment:
Brackets are used only for tuples. You can remove them

Reply:
Fixed.